R/exploratoryfactoranalysis.R

```diff
@@ -451,7 +451,7 @@ ExploratoryFactorAnalysis <- function(jaspResults, dataset, options, ...) {
     tp <- rep(c(gettext("Data"), gettext("Simulated data from parallel analysis")), each = n_col)
 
   } else { # do not display parallel analysis
-    evs <- eigen(cov(dataset, use = "pairwise.complete.obs"), only.values = T)$values
+    evs <- eigen(cor(dataset, use = "pairwise.complete.obs"), only.values = T)$values
```

Comment:
Can you please change T to TRUE? Just to avoid unintended side effects?

Reply:
yes, sorry, my bad